### DISCUSSION TITLE
Welcome to recent-activity Discussions!

### DISCUSSION THREAD

**Discussion by PuneetGopinath** [Readme-Workflows/recent-activity#1] [- News]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PuneetGopinath** [comment]
I am Puneet Gopinath from India
Am this repo's collaborator mainly helping on core node.js

**Comment by Andre601** [comment]
I'm Andre.
If this was a company would I probably be "Head of Documentation" or something... I mainly do work on the various Markdown files (README, CHANGELOG, ...) and the Wiki of this project since documentation is my passion.

**Comment by abhijoshi2k** [comment]
I am Abhishek Joshi
I work on functionality of this workflow.
Adding new features or fixing bugs is my task here!

**Comment by Andre601** [comment]
Hey everyone.

I want to quickly apologize for my reactions in #54 (Namely [this comment](https://github.com/Readme-Workflows/recent-activity/pull/54#issuecomment-846429598)).
I was angry at that time because changes were made without my knowledge and without discussing it first while I was a codeowner there.

While my situation there should be understandable do I get that it wasn't acceptable, so I apologize now in this comment. I should've shown a more professional behaviour and moved this to (at most) private chats with the respective people.

I hope we can forget this event and move on stronger and better than before.
